src/assets/constants/flex.ts

```diff
@@ -0,0 +1,16 @@
+export const ALIGNMENTS = {
+  center: "center",
+  start: "flex-start",
+  end: "flex-end",
+  baseline: "baseline",
+  stretch: "stretch",
+};
+
+export const JUSTIFY_CONTENT = {
+  center: "center",
+  start: "flex-start",
+  end: "flex-end",
+  between: "space-between",
+  around: "space-around",
+  evenly: "space-evenly",
+};
```

Comment:
you dont need it
we use consts usually when we want a field to be responsive or its too complicated to understand